aviary/subsystems/mass/flops_based/furnishings.py

```diff
@@ -32,21 +33,14 @@ def setup(self):
     def setup_partials(self):
         self.declare_partials(of=Aircraft.Furnishings.MASS, wrt='*')
 
-    def compute(
-        self, inputs, outputs, discrete_inputs=None, discrete_outputs=None
-    ):
-        aviary_options: AviaryValues = self.options['aviary_options']
-
-        flight_crew_count = aviary_options.get_val(Aircraft.CrewPayload.NUM_FLIGHT_CREW)
-        first_class_count = aviary_options.get_val(Aircraft.CrewPayload.NUM_FIRST_CLASS)
-
-        business_class_count = aviary_options.get_val(
-            Aircraft.CrewPayload.NUM_BUSINESS_CLASS)
+    def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
```

Comment:
They technically can be removed. I remember having a philosophical debate about them with DJ, where his argument was that, while they are unused optional kwargs, since they are part of the call, they should be included in the signature. 

Right now in Aviary, we include them in 107 of the 389 calls to compute, even though we never use the discrete variable capability of openmdao.